tools/clang/test/CodeGenSPIRV/rich.debug.debugscope.hlsl

```diff
@@ -46,7 +43,8 @@ float4 main(float4 color : COLOR) : SV_TARGET
 // CHECK:      %if_true = OpLabel
 // CHECK-NEXT: {{%\d+}} = OpExtInst %void [[set]] DebugScope [[ifStmtLexBlock]]
       c = c + c;
-// CHECK: {{%\d+}} = OpExtInst %void [[set]] DebugScope [[tempLexBlock]]
+// TODO: It has to generate
```

Comment:
Can this be done in this PR? It might get forgotten here.

Reply:
done